packages/components/app/styles/components/pagination.scss

```diff
@@ -10,19 +10,41 @@
 
 .hds-pagination {
   display: grid;
-  grid-template-areas: "left center right";
+  grid-template-areas: "info nav selector";
   grid-template-rows: auto;
   grid-template-columns: 1fr auto 1fr;
   align-items: center;
   margin: 0 auto;
 }
 
+// Responsive layout
+// max-width is based on where component layout potentially breaks down
+@media screen and (max-width: 1000px) {
+  .hds-pagination {
+    grid-template-areas:
+      "nav"
+      "info"
+      "selector";
+    grid-template-rows: auto auto auto;
+    grid-template-columns: 1fr;
+    row-gap: 8px;
```

Comment:
This value can be adjusted if desired and can be exposed as a component-level token (CSS custom property) before release.